pyfar/io/_codec.py

```diff
@@ -154,7 +154,7 @@ def _decode(obj, zipfile):
     if isinstance(obj, dict):
         for key in obj.keys():
             _inner_decode(obj, key, zipfile)
-    elif any([isinstance(obj, x) for x in [list, tuple, set, frozenset]]):
+    elif type(obj) in [list, tuple, set, frozenset]:
```

Comment:
Is using `type` instead of `isinstance` intended? If not `isinstance(obj, (list, tuple, set, frozenset))` should also work.